.env.example

```diff
@@ -0,0 +1,6 @@
+
+PG_HOST=127.0.0.1
+PG_PORT=45775
```

Comment:
move PG_PORT to constants.js where it will default to 45775 or the value given in .env